groups/data-lake/vault/approle

```diff
@@ -0,0 +1,20 @@
+variable "hashicorp_vault_role_id" {
+  description = "Your Hashicorp Vault role id"
+  type = string
+}
+
+variable "hashicorp_vault_secret_id" {
+  description = "Your Hashicorp Vault secret id"
+  type = string
+}
```

Comment:
Can we make the purpose of these variables a little clearer? I'm thinking something along the lines of:

`hashicorp_vault_role_id `:
 `The role identifier used when retrieving configuration from Hashicorp Vault`

`hashicorp_vault_secret_id`:
 `The secret identifier used when retrieving configuration from Hashicorp Vault`

If we do make changes, could we also update the same variables descriptions in `groups/data-lake/vault/userpass`?